cloudfoundry-operations/src/main/java/org/cloudfoundry/operations/domains/DefaultDomains.java

```diff
@@ -58,6 +61,22 @@ public DefaultDomains(CloudFoundryClient cloudFoundryClient) {
             .after();
     }
 
+    @Override
+    public Flux<Domain> list(ListDomainsRequest request) {
+        return ValidationUtils
+            .validate(request)
+            .flatMap(request1 -> getDomains(this.cloudFoundryClient, request1))
+            .map(resource -> toDomain(resource));
+    }
+
+    private static Flux<DomainResource> getDomains(CloudFoundryClient cloudFoundryClient, ListDomainsRequest request) {
```

Comment:
@LokeshN We isolate methods that make network calls, as you've done here, but by convention we'd call them what they were requesting.  In this case it would be `requestListDomains()`.  In a full implementation (see my comment on the main PR) we'd expect `requestListPrivateDomains()` and `requestListSharedDomains()`.